packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -330,8 +344,22 @@ pc.createOffer = (peerConnection, {meetingId, remoteQualityLevel}) => {
         });
         return peerConnection;
       })
-      .catch((err) => {
-        LoggerProxy.logger.error(`Peer-connection-manager:index#createOffer --> ${err}`);
+      .catch((error) => {
+        LoggerProxy.logger.error(`Peer-connection-manager:index#createOffer --> ${error}`);
+
+        const metricName = METRICS_OPERATIONAL_MEASURES.PEERCONNCTION_FAILURE;
+        const data = {
+          correlation_id: meetingId,
+          reason: error.message,
```

Comment:
Same as above

Reply:
This is missing the Locus ID field, is it not necessary for this metric?

Reply:
not needed here correlation ID should be sufficient , We could have same locus id for two meeting joining one after other